Integrations/integration-CrowdStrikeHost.yml

```diff
@@ -94,6 +94,7 @@ script:
     };
 
     var uploadIOC = function() {
+        args.expiration_days = Number(args.expiration_days)
```

Comment:
parseInt?

Reply:
not sure it makes a difference, but fixed...